cookbooks/fb_systemd/README.md

```diff
@@ -22,7 +22,8 @@ Attributes
 * node['fb_systemd']['logind']['enable']
 * node['fb_systemd']['logind']['config']
 * node['fb_systemd']['networkd']['enable']
-* node['fb_systemd']['resolved']['enable']
+* node['fb_systemd']['networkd']['enable']
+* node['fb_systemd']['resolved']['enable-nss-resolve']
```

Comment:
I believe the convention is to use underscores instead of dashes for these things (i.e. `enable_nss_resolve`)